stdlib/public/libexec/swift-backtrace/main.swift

```diff
@@ -895,7 +895,7 @@ Generate a backtrace for the parent process.
     }
 
     if args.threads! {
-      for (ndx, thread) in target.threads.enumerated() {
+      for (ndx, thread) in target.threads.enumerated().sorted(by: { t, _ in t.0 == target.crashingThreadNdx}) {
         dump(ndx: ndx, thread: thread)
       }
     } else {
```

Comment:
Rather than using `sorted(by:)`, particularly as the list is already sorted aside from the one element we want at the top, why don't we just make the code from the `else` run unconditionally, i.e.

```suggestion
    dump(ndx: target.crashingThreadNdx, thread: crashingThread)
    if args.threads! {
      for (ndx, thread) in target.threads.enumerated() {
        if (ndx != target.crashingThreadNdx) {
          dump(ndx: idx, thread: thread)
        }
      }
    }
```